### PR TITLE
wrong credentials error message showing

### PR DESCRIPTION
**Pull request by AtmegaBuzz**
Issue :  Wrong login credentials (Validation Message) not showing. #889 

prev it was not showing error message
image

### CODE REVIEW

**Comment by fredfalcon** [comment]
I think the message is there we just need to show it.  We don’t need another view

**Comment by AtmegaBuzz** [comment]
ok if thats the case then its not passing it in messages in the request. ill check

**Comment by AtmegaBuzz** [comment]
@fredfalcon now error is showing without custom view 
image

File: website/views.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #from django_cron import CronJobBase, Schedule
 from allauth.account.models import EmailAddress
 from allauth.account.signals import user_logged_in
+from allauth.account.views import LoginView,_ajax_response
```
Comment (fredfalcon) [inline website/views.py:23]: Can remove this

**Review by fredfalcon — CHANGES_REQUESTED** [review]
See comment

File: website/views.py (hunk 1 above)
Comment (AtmegaBuzz) [reply website/views.py]: ok

**Comment by AtmegaBuzz** [comment]
@fredfalcon resolved

**Review by fredfalcon — APPROVED** [review]

**Comment by fredfalcon** [comment]
Thank you!

**Comment by fredfalcon** [comment]
If you link the issue to the pr it will close automatically https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Pull request merged**